board-viewer.html

```diff
@@ -516,10 +516,6 @@
       }
     }
 
-    if (!component) {
```

Comment:
It looks like when you committed d68a398cff, you did it from an older local copy of board-viewer.html; your patch is undoing commits b691fcd6 and 6d3fd7bf3.

One way to fix is for you to roll back your changes to commit 6d3fd7bf, then apply your changes as a patch, then use `git add -i` to select which parts should *actually* be included in the commit, then resubmit a new branch.

There are probably a dozen other ways to do the above too... in the end though, we just want the commit to contain your intended changes :)